presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -404,6 +412,38 @@ public void setJmxExport(boolean export)
         jmxExportListener.accept(this, export);
     }
 
+    @Override
+    public Duration getQueuedTimeout()
+    {
+        synchronized (root) {
+            return queuedTimeout;
+        }
+    }
+
+    @Override
+    public void setQueuedTimeout(Duration queuedTimeout)
+    {
+        synchronized (root) {
+            this.queuedTimeout = queuedTimeout;
+        }
+    }
+
+    @Override
+    public Duration getRunningTimeout()
```

Comment:
I feel like these should be called queueTimeLimit and runningTimeLimit instead. Timeout has a slightly different meaning.